src/overlays/actors/ovl_Arrow_Fire/z_arrow_fire.c

```diff
@@ -156,12 +159,12 @@ void ArrowFire_Fly(ArrowFire* this, GlobalContext* globalCtx) {
     // copy position and rotation from arrow
     this->actor.world.pos = arrow->actor.world.pos;
     this->actor.shape.rot = arrow->actor.shape.rot;
-    distanceScaled = Math_Vec3f_DistXYZ(&this->unkPos, &this->actor.world.pos) * (1.0f / 24.0f);
-    this->unk_158 = distanceScaled;
+    distanceScaled = Math_Vec3f_DistXYZ(&this->trailEnd, &this->actor.world.pos) * (1.0f / 24.0f);
+    this->trailLen = distanceScaled;
```

Comment:
I think we can afford the full word :p
```suggestion
    this->trailLength = distanceScaled;
```